packages/server/src/errors/http-error.js

```diff
@@ -0,0 +1,15 @@
+export default class HttpError extends Error {
+  static _400(content) {
```

Comment:
The `_` prefix is new to me—I think we can just use numbers as method names?

But otherwise, I wonder what you think about naming these errors based on their status codes, e.g. naming them similar to (and perhaps also using the constants from) [http-status-codes](https://www.npmjs.com/package/http-status-codes)?

Reply:
I think you cannot use a method/function that starts with a number.

I like the idea of using status codes, would be more readable.